contracts/ISSVNetwork.sol

```diff
@@ -131,6 +131,11 @@ interface ISSVNetwork {
         Pod pod
     );
 
+    event FeeRecipientAddressAdded(
+        address ownerAddress,
```

Comment:
@vadiminc @mtabasco Should this be indexed address?

Reply:
It depends on the use case. If we want to allow search using `ownerAddress`, `recipientAddress` or both, yes, then should be indexed